### ISSUE TITLE
Move NUM_IMAGES from kernel.json to Dockerfile

### ISSUE THREAD

**Issue by 3Dcube** [sourceryinstitute/jupyter-CAF-kernel#1]
I want to change ```%num_images``` default value. My idea was to change environment variable ```NUM_IMAGES```
I just type:
```bash
docker run -e NUM_IMAGES=8 -i -t -p 8888:8888 sourceryinstitute/jupyter-caf-kernel:latest
```
But it didn't work because ```kernel.json``` override it.
So my suggestion is to move variable declaration from ```kernel.json``` to ```Dockerfile```, after that everyone can change it easily within one docker command.

**Comment by zbeekman** [comment]
Sure, I can do that! TBH, my kernel is super hack-y and I would love any and all help and suggestions that I can get! I think this is a good idea.

Just FYI, if/when I find time, I'm going to create a docker image over at https://github.com/sourceryinstitute/OpenCoarrays that will rebuild every time code is pushed to master, since master is (intended to be) pretty stable. This was people can try/submit bugs without having to install master themselves. I'll probably implement your suggestion much sooner though.

**Comment by zbeekman** [comment]
Fixed by #2. Thanks for your contribution @3Dcube :tada: :100:
